server/analysis/tests/simulation.py

```diff
@@ -0,0 +1,145 @@
+#
+# OtterTune - simulation.py
```

Comment:
Yes I plan to add simulation for GPR/NN.
Also, I want to simulate different workloads.
I prefer not include simulation in tests, because its purpose is to try different hyperparameters and make comparisons.